odd-platform-ui/src/components/Management/LabelsList/EditableLabelItem/EditableLabelItem.tsx

```diff
@@ -5,12 +5,12 @@ import EditIcon from 'components/shared/Icons/EditIcon';
 import DeleteIcon from 'components/shared/Icons/DeleteIcon';
 import ConfirmationDialog from 'components/shared/ConfirmationDialog/ConfirmationDialog';
 import AppButton from 'components/shared/AppButton/AppButton';
-import LabelEditFormContainer from '../LabelEditForm/LabelEditFormContainer';
+import LabelEditForm from '../LabelEditForm/LabelEditForm';
 import * as S from './EditableLabelItemStyles';
 
 interface EditableLabelItemProps {
   label: Label;
-  deleteLabel: (params: LabelApiDeleteLabelRequest) => Promise<void>;
+  deleteLabel: (params: LabelApiDeleteLabelRequest) => Promise<unknown>;
```

Comment:
The same thing. 
Now this component isn't shared and it uses only in one place. So now we don't need to do it reusable, so we could don't use this prop. Just import `deleteLabelItem` and dispatch it where you need